packages/rancher-turtles/rancher-turtles/package.yaml

```diff
@@ -0,0 +1,2 @@
+url: https://github.com/rancher-sandbox/rancher-turtles/releases/download/v0.5.0/rancher-turtles-0.5.0.tgz
+version: 100.1.0
```

Comment:
Please see the following rules for versioning: https://github.com/rancher/charts/blob/dev-v2.8/README.md#versioning-charts

You are trying to merge at `dev-v2.8` branch.